### DISCUSSION TITLE
电信宽带国外访问被严重限速

### DISCUSSION THREAD

**Discussion by malagebidi** [XTLS/Xray-core#2871] [General]
使用VLESS-Vision-REALITY偷别人配置，在线ping测得节点平均ping值为50ms，结果我本机ping都是在140ms以上，实际使用效果更是稀烂。

PS：刚开始使用的前几天没问题。

有没有一样情况的？

**Comment by Kbstsn** [comment]
电信的话买台CN2 GIA吧。有点网络常识就知道怎么回事了

**Reply by malagebidi** [reply]
恩我试了一些，目前在用cn2gia和9929，就是QoS了以后享受不了低延迟，可惜。

**Reply by Kbstsn** [reply]
我电信也是被逼着买了台CN2 GIA
